presto-main/src/main/java/com/facebook/presto/server/remotetask/HttpRemoteTask.java

```diff
@@ -234,6 +230,8 @@ public final class HttpRemoteTask
     private final DecayCounter taskUpdateRequestSize;
     private final SchedulerStatsTracker schedulerStatsTracker;
 
+    private final EventLoop taskEventLoop;
```

Comment:
- Atomic fields no longer need to be atomic (`nextSplitId`, `needsUpdate`, `sendPlan`, `started`, `aborted`, `outputBuffers`). It is better to remove the atomic as it may suggest that the variable is accesses from multiple threads concurrently.
- Remove `GuardedBy` annotations, as the accesses are no longer protected by a mutex. Instead you may want to add a comment at the beginning of the class and explain what is going on in this class (e.g.: that all interactions with the TaskState have to be done via an event loop)

Reply:
Good point. Updating the code.